constraint/pkg/client/client.go

```diff
@@ -455,7 +493,13 @@ func (c *client) validateConstraint(constraint *unstructured.Unstructured, lock
 // ValidateConstraint returns an error if the constraint is not recognized or does not conform to
 // the registered CRD for that constraint.
 func (c *client) ValidateConstraint(ctx context.Context, constraint *unstructured.Unstructured) error {
-	return c.validateConstraint(constraint, true)
+	return c.validateConstraint(constraint, true, constraintGroup)
+}
+
+// ValidateMutation returns an error if the constraint is not recognized or does not conform to
+// the registered CRD for that constraint.
+func (c *client) ValidateMutation(ctx context.Context, constraint *unstructured.Unstructured) error {
```

Comment:
Do you plan to make ValidateMutation do something different than ValidateConstraint in the future?

Reply:
Discussed offline.